projects/Mallard/src/screens/settings-screen.tsx

```diff
@@ -48,6 +48,9 @@ const MiscSettingsList = React.memo(
                 onPress: onChange,
                 proxy: (
                     <Switch
+                        accessible={True}
```

Comment:
True should be lowercase  (`true`) in javascript (confusing because it's upeprcase in python)

Reply:
Thanks, Pip. That's True!